crates/air-parser/src/parser/lexer/air_lexer.rs

```diff
@@ -194,6 +195,25 @@ fn string_to_token(input: &str, start_pos: usize) -> LexerResult<Token> {
     }
 }
 
+fn parse_last_error(input: &str, start_pos: usize) -> LexerResult<Token<'_>> {
+    let last_error_size = LAST_ERROR.len();
+    let last_error_path = match &input[last_error_size..] {
```

Comment:
Maybe also account for possible `$`? Like `%last_error%.$.instruction`

Reply:
Hmm, I am not sure, we're going to get rid of json path in the nearest future.

Reply:
Discuss it in PM and decided to use `.$` here for consistency.